### ISSUE TITLE
Phone number validation

### ISSUE THREAD

**Issue by fgouget** [osm-fr/osmose-backend#227]
In France a lot of internationalized phone numbers are wrong because they don't omit the initial zero.
That one will have 'phone=+33 01 02 03 04 05' instead of the correct  'phone=+33 1 02 03 04 05'.
The extra 0 means the number is not usable.

Similarly a number of short numbers have been internationalized and thus are broken. The typical example if '+33 3631'.

So I propose a new Osmose plugin (attached below) to check that phone numbers are valid. Similar issues may exist in other countries but I'm not familiar with their phone numbering schemes so for now it only works for France. Notes:
* I don't have the setup required for testing the new plugin so it's totally untested.
* I picked an issue id somewhat randomly. It may need to be adjusted.
* The plugin can also internationalize regular French phone number. This will likely generate a ton of reports so maybe this is not a good idea.

[Phone_fr.py](https://github.com/osm-fr/osmose-backend/files/1258928/Phone_fr.txt)

**Comment by frodrigo** [comment]
https://fr.wikipedia.org/wiki/Num%C3%A9ro_de_t%C3%A9l%C3%A9phone

**Comment by frodrigo** [comment]
I pick up and fix your commit. Please fell free to continue to ammend it and make pull request.

**Comment by frodrigo** [comment]
http://osmose.openstreetmap.fr/fr/errors/?item=3092&useDevItem=all
@fgouget can you review the first results, please?

**Comment by fgouget** [comment]
The ones I have seen look ok.
Notes:
* I mostly saw 'Missing international prefix' errors but I did find a couple of '« 0 » en trop après le préfixe international et le code pays' errors (Médiathèque André Cancelier de Sannois). All was fine.
In my tree I had a patch to not add the international prefix to numbers starting with 09 but I think that was overly cautious: these numbers are used for triple-play service and can be called from abroad so adding the prefix should be fine. It's only the 08 we need to avoid.
* I saw one emergency_phone=0102030405 on a fire station that did not get fixed. However I have not seen any documentation for that tag (it's different from both amenity=emergency_phone and emergency=phone) and it's probably not very common. So it may not make sense to add support for it.
* In the case where with have 'phone=0102030405', so without any space, it could make sense to propose 'phone=+33 1 02 03 04 05' so adding the usual French phone number formatting. But that's probably overkill and could even be controversial. So I'm fine either way.
